### ISSUE TITLE
What's the learning rate during the training stage of subject-512?

### ISSUE THREAD

**Issue by thulishuang** [Yuanshi9815/OminiControl#98]
<img width="457" alt="Image" src="https://github.com/user-attachments/assets/04f6b99f-78b9-4638-98f9-ef67b4b10ad0" />

I would like to know the learning rate for training the 512 subject model. It is set to 1 in the config yaml file, but what is the actual learning rate used during training?

**Comment by Yuanshi9815** [comment]
Hi @thulishuang ,

You can refer to the paper of Prodigy optimizer [here](https://arxiv.org/abs/2306.06101).

The optimizer of AdamW with `lr=1e-4` also works well.

:D
